src/felles/stønadstype.ts

```diff
@@ -6,7 +6,7 @@ export enum Stønadstype {
     DAGLIG_REISE_TSR = 'DAGLIG_REISE_TSR',
 }
 
-const stønadstypeInterntVedtakTittel: Record<Stønadstype, string> = {
+export const stønadstypeTilTeXt: Record<Stønadstype, string> = {
```

Comment:
Ville kanskje laget en egen funksjon for stønadstype til tekst, sånn at man kan unngå stor bokstav midt i en setning når det brukes i aktivitet:
<img width="1150" height="556" alt="image" src="https://github.com/user-attachments/assets/da713038-4d83-45a6-95f7-48db1b836165" />